### ISSUE TITLE
IDLE: Run ... Customized strips backslashes from arguments

### ISSUE THREAD

**Issue by davidyorke** [python/cpython#93016]
### PART 1/2
**Actions**
In editor, do "Run... Customized"
In text box, Paste or type a valid Windows path (e.g.: `c:\Users`, `\\server\share`)
Select "OK"

**Expected response:**
script receives argument as entered without modification just as it does when run on the command line (e.g.: `python prog.py c:\users`)

**Actual response:**
script receives argument with backslashes treated as escape characters (e.g.: `c:users`, `\servershare`) 

**Workaround:**
argument is not altered if entered in quotes.  This is not the same behavior as the command line where quotes are accepted as part of the argument.

### Part 2/2
**Actions**
In editor, do "Run... Customized"

**Expected response:**
text box is populated with previous entry, a legal windows path (e.g. `c:\Users)

**Actual response:**
text box is populated with previous entry mangled as above surrounded by curly braces (e.g.: `{c:users}`, `{\serverpath}`).   If workaround above is used, then the string is correct, but quotes are gone and curly braces are present (e.g.: `{c:\users}`, `{\\server\share}`

### System information
Windows 10 Home x86-64 (21H1 build 19043.1706)
Idle version 3.10.4

### Linked PRs
* gh-157635
* gh-158049
* gh-158050

**Comment by serhiy-storchaka** [comment]
Both parts are fixed in python/cpython#157635. On Windows, the arguments are now split by the same rules as `python.exe` splits the command line, so backslashes are literal, and quotes work as on the command line. The previous arguments are now displayed quoted and joined, so they can be edited and are parsed back to the same list.

**Comment by beta-tester** [comment]
is this the same issue?

for the issue with “Run… Customized” i tried
the simple test script:
```py
import sys; print(sys.argv[1:])
```

at first “Run…Customized”, i typed in:
```sh
xyz "C:\abcd feg\123.txt"
```

at second “Run…Customized”, the previous entered text altered to:
```sh
xyz {C:\abcd feg\123.txt}
```

at third “Run…Customized”, the previous altered text altered to:
```sh
xyz \{C:abcd feg123.txt\}
```

at that point the text does not alter anymore.

the script output of the runs above
at first run:
```py
['xyz', 'C:\\abcd feg\\123.txt']
```

at second run:
```py
['xyz', '{C:abcd', 'feg123.txt}']
```

at third run:
```py
['xyz', '{C:abcd', 'feg123.txt}']
```

i am using Python IDLE 3.14.7 for windows / Python 3.14.7 with td/tk 9.0.4
installed with `winget install Python.Python.3.14` on windows 11 25h2

**Comment by dqqFA8995** [comment]
Thank you for the information.

**Comment by serhiy-storchaka** [comment]
Yes, it is the same issue. It was fixed in #157635 after 3.14.7 was released; the fix will be in the next 3.14 and 3.13 bugfix releases.
